doc/Language/module-packages.pod6

```diff
@@ -0,0 +1,180 @@
+=begin pod :tag<perl6>
+
+=TITLE Module Packages
+
+=SUBTITLE Creating module packages for code reuse.
+
+
+I<N.B.> "Module" is an overloaded term in Perl 6; this document
+focuses on use of the C<module> declarator.
+
+
+=head1 What are modules?
+
+Modules, like classes and grammars, are a kind of
+L<package|/language/packages>. Fundamentally, they are metaclasses
```

Comment:
Modules are not metaclasses. They are named containers for code and data. The module object itself is just an instance of the ModuleHOW metaclass.

Reply:
Thanks; I've pushed 71cf557 to clarify this point. Better?